doc/source/user_guide/text.rst

```diff
@@ -6,8 +6,71 @@
 Working with text data
 ======================
 
+.. _text.types:
+
+Text Data Types
+---------------
+
+.. versionadded:: 1.0.0
+
+There are two main ways to store text data
+
+1. ``object`` -dtype NumPy array.
+2. :class:`TextDtype` extension type.
+
+We recommend using :class:`TextDtype` to store text data.
+
+Prior to pandas 1.0, ``object`` dtype was the only option. This was unfortunate
+for many reasons:
+
+1. You can accidentally store a *mixture* of strings and non-strings in an
+   ``object`` dtype array. It's better to have a dedicated dtype.
+2. ``object`` dtype breaks dtype-specific operations like :meth:`DataFrame.select_dtypes`.
+   There isn't a clear way to select *just* text while excluding non-text
+   but still object-dtype columns.
+3. When reading code, the contents of an ``object`` dtype array is less clear
+   than ``text``.
+
+Currently, the performance of ``object`` dtype arrays of strings and
+:class:`arrays.TextArray` are about the same. We expect future enhancements
+to significantly increase the performance and lower the memory overhead of
+:class:`~arrays.TextArray`.
+
+.. warning::
+
+   ``TextArray`` is currently considered experimental. The implementation
+   and parts of the API may change without warning.
+
+For backwards-compatibility, ``object`` dtype remains the default type we
+infer a list of strings to
+
+.. ipython:: python
+
+   pd.Series(['a', 'b', 'c'])
+
+To explicitly request ``text`` dtype, specify the ``dtype``
+
+.. ipython:: python
+
+   pd.Series(['a', 'b', 'c'], dtype="text")
+   pd.Series(['a', 'b', 'c'], dtype=pd.TextDtype())
+
+Or ``astype`` after the ``Series`` or ``DataFrame`` is created
+
+.. ipython:: python
+
+   s = pd.Series(['a', 'b', 'c'])
+   s
+   s.astype("text")
+
+Everything that follows in the rest of this document applies equally to
+``text`` and ``object`` dtype.
+
 .. _text.string_methods:
 
+String Methods
```

Comment:
Text Methods? (actually now that we are calling this text, is that weird to have a .str accessor?)

Reply:
I prefer String methods as many of these are aliases to methods in the `string` module / `str` objects.